helpers/index.ts

```diff
@@ -171,6 +171,11 @@ export const installTemplate = async (
         );
       }
     }
+
+    // Create tool-output directory
+    if (props.tools && props.tools.length > 0) {
```

Comment:
i see that python tool is generating the output dir if it doesn't exist - how about we add this to TS too?

Reply:
I think this change is already applied to TS.